src/display/metadata.js

```diff
@@ -103,10 +103,31 @@ class Metadata {
       }
 
       for (let j = 0, jj = desc.childNodes.length; j < jj; j++) {
-        if (desc.childNodes[j].nodeName.toLowerCase() !== "#text") {
-          const entry = desc.childNodes[j];
+        const entry = desc.childNodes[j];
+        if (entry.nodeName.toLowerCase() !== "#text") {
```

Comment:
Given the overall indentation level and the size of the code below, things would be slightly more readable with this condition inverted like so:
```suggestion
        if (entry.nodeName.toLowerCase() === "#text") {
          continue;
        }
```